C01/project-metadata/C01-ProjectId612-metadata.json

```
@@ -69,12 +69,44 @@
       }
     },
     "type": "Feature",
+    "place_name": "Taita–Taveta, Kenya",
```

```suggestion
    "properties": {
      "name": "Taita–Taveta, Kenya Test",
    }
```

suggesting this change based on [the Feature part of the geojson spec][1] and [this example here][2].

[1]: https://datatracker.ietf.org/doc/html/rfc7946#section-3.2
[2]: https://macwright.com/2015/03/23/geojson-second-bite.html#features